drivers/usb/device/usb_dc_stm32.c

```diff
@@ -67,6 +80,23 @@
  */
 #define NUM_BIDIR_EP NUM_OUT_EP
 
+#ifdef USB
```

Comment:
Yes, it is define in soc header file in stm32cube HAL. We could use the DT generated ST_STM32_OTGFS_50000000_BASE_ADDRESS and ST_STM32_USB_40005C00_BASE_ADDRESS, but I don't like it because it contains the memory address.